bad_actors/flask_api/unit_tests/flask_api_unittests.py

```diff
@@ -0,0 +1,17 @@
+import unittest, requests
```

Comment:
Please see example format in:
bad_actors/preprocessing_tools/unit_tests/test_rank_app_importer.py